packages/google_maps_flutter/lib/src/circle_updates.dart

```diff
@@ -36,9 +36,15 @@ class _CircleUpdates {
         .map(idToCurrentCircle)
         .toSet();
 
+    bool currentNotEqualsPrevious(Circle _current) {
```

Comment:
1. For parameters, we don't need to use `_` prefix as they are already scoped. Can we change this to `Circle current`. 
2. I think the method reads better if we call it: `hasChanged(Circle current)`.
3. Add dart-docs for this method, something along the lines of what we compare it against, etc.

Note, this applies for all Polygon/Polyline/Markers etc.

Reply:
Yeah, hasChanged is a better name. I'll change it.